.gitignore

```diff
@@ -1,2 +1,5 @@
 node_modules
-.DS_Store
\ No newline at end of file
+.DS_Store
+.nyc_output
+coverage
+node_modules_bak
```

Comment:
Should this be here? Seems like it might have been included accidentally.

Reply:
Ah, yes, because the cost-of-modules takes quite some time i cancelled it and it left this folder. I added it in the ignore as i figured this shouldn't be checked in. I will remove it from the gitignore ;-)